src/lib/accessibility/Behaviors/Toolbar/ToolbarBehavior.ts

```diff
@@ -0,0 +1,13 @@
+import { Accessibility } from '../../interfaces'
+
+const ToolbarBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: 'toolbar',
+      'aria-label': props['aria-label'],
+      'aria-disabled': !!props['disabled'],
```

Comment:
same here regarding exclamation marks

Reply:
similar to tab behavior. wouldn't the screen reader with this read 'not disabled' each time user navigates to a button? note that different readers might read it differently.

Reply:
will remove the double excl marks if we decide to be consistent with that approach.